### PR TITLE
8.10.1 Release notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/3960

Preview:
- [Index page](https://security-docs_3962.docs-preview.app.elstc.co/guide/en/security/master/release-notes.html)
- [Release notes](https://security-docs_3962.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.10.1.html#release-notes-8.10.1)

### CODE REVIEW

**Comment by natasha-moore-elastic** [comment]
Just checking if we still want to document [this](https://github.com/elastic/security-docs/issues/3822#issuecomment-1717600089) in the release notes?

**Comment by nastasha-solomon** [comment]
Thanks for following up on this, @natasha-moore-elastic. @kevinlog @tomsonpl did you want to release note https://github.com/elastic/kibana/pull/166178 as a new feature or enhancement in 8.10.1?

**Comment by tomsonpl** [comment]
Hey, I would say we don't have to specify this in the release notes. Just updating the docs would be fine. But, what's your opinion on this @kevinlog ?

**Comment by kevinlog** [comment]
> Hey, I would say we don't have to specify this in the release notes. Just updating the docs would be fine

Yes, this sounds good to me

File: docs/release-notes/8.10.asciidoc [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
 
+[discrete]
+[[release-notes-8.10.1]]
+=== 8.10.1
+
+[discrete]
+[[bug-fixes-8.10.1]]
+==== Bug fixes
+
+* Updates the Elastic AI Assistant response schema ({pull}166300[#166300]).
```
Comment (stephmilovic) [inline docs/release-notes/8.10.asciidoc:12]:
```suggestion
* Updates the Elastic AI Assistant response schema ({pull}166300[#166300], {pull}166495[#166495]).
```
I forgot something in the first PR so there were actually 2!

**Review by stephmilovic — APPROVED** [review]
Needs additional PR link, other than that LGTM! Thanks @nastasha-solomon

**Review by natasha-moore-elastic — APPROVED** [review]

**Review by jmikell821 — APPROVED** [review]
Good to merge, thanks!

**Pull request merged**